src/api/fship_app/views.py

```diff
@@ -374,6 +374,93 @@ def getConnectedUsers(request, id):
 
     return HttpResponse(json.dumps(response), content_type='application/json; charset=UTF-8')
 
+## Get similar users from the same country
+def getSimilarUsersByCountry(request, id):
+    ## Start the session
+    session = driver.session()
+    # Define the query
+    query = f"""
+    MATCH (u:User) - [:comesFROM] -> (c:Country) 
+    WHERE id(c) = {id} 
+    RETURN id(u) AS id, u.name AS name, u.email AS email, u.github AS github
+    """ 
+
+    result = session.run(query) ## Execute the function 
+    ## The result is not subsciptible so we loop to get the single value 
+
+    users = [
+        models.FShipUser(
+            id=record["id"], 
+            name=record["name"], 
+            gitHandle=record["github"], 
+            email=record["email"]).toJSON()
+            for record in result
+    ]
+
+    response = {
+        "users" : users
+    }
+
+    return HttpResponse(json.dumps(response), content_type='application/json; charset=UTF-8')
+
+## Get similar users with the same bio
+def getSimilarUsersByBio(request, id):
```

Comment:
Um, what this will do is basically return all users who have a bio with a particular id (exactly same bio). In a real scenario, the bio will be different for each user, so we need to check for keywords and then return the one with most common ones. Since this will take a lot of time to implement properly, we're planning to scrap this endpoint for now.

Reply:
okay. I will scrap it.

Reply:
@akshgpt7  I have scrapped getSimilarUsersByBio()